internal/operators/ocs/validations.go

```diff
@@ -11,29 +11,36 @@ import (
-	OCSMinimumRAMGiB                  int64             `envconfig:"OCS_MINIMUM_RAM_GiB" default:"57"`
-	OCSRequiredDisk                   int64             `envconfig:"OCS_MINIMUM_DISK" default:"3"`
-	OCSRequiredDiskCPUCount           int64             `envconfig:"OCS_REQUIRED_DISK_CPU_COUNT" default:"2"` // each disk requires 2 cpus
-	OCSRequiredDiskRAMGiB             int64             `envconfig:"OCS_REQUIRED_DISK_RAM_GiB" default:"5"`   // each disk requires 5GiB ram
-	OCSRequiredHosts                  int64             `envconfig:"OCS_MINIMUM_HOST" default:"3"`
-	OCSStandardAndCompactModeCPUCount int64             `envconfig:"OCS_STANDARD_AND_COMPACT_MODE_CPU_COUNT" default:"30"` // Compact Mode cpu requirements for 3 nodes( 4*3(master)+6*3(OCS) cpus)
-	OCSStandardAndCompactModeRAMGiB   int64             `envconfig:"OCS_STANDARD_AND_COMPACT_MODE_RAM_GiB" default:"81"`   // Compact Mode ram requirements (8*3(master)+57(OCS) GiB)
-	OCSMasterWorkerHosts              int64             `envconfig:"OCS_REQUIRED_MASTER_WORKER_HOSTS" default:"6"`
-	OCSDisksAvailable                 int64             `envconfig:"OCS_DISKS_AVAILABLE" default:"3"`
-	OCSDeploymentType                 ocsDeploymentMode `envconfig:"OCS_DEPLOYMENT_TYPE" default:"None"`
+	OCSRequiredDisk                       int64             `envconfig:"OCS_MINIMUM_DISK" default:"3"`
+	OCSDisksAvailable                     int64             `envconfig:"OCS_DISKS_AVAILABLE" default:"3"`
+	OCSRequiredDiskCPUCount               int64             `envconfig:"OCS_REQUIRED_DISK_CPU_COUNT" default:"2"` // each disk requires 2 cpus
+	OCSRequiredDiskRAMGiB                 int64             `envconfig:"OCS_REQUIRED_DISK_RAM_GiB" default:"5"`   // each disk requires 5GiB ram
+	OCSRequiredHosts                      int64             `envconfig:"OCS_MINIMUM_HOST" default:"3"`
+	OCSCompactModeClusterCPUCount         int64             `envconfig:"OCS_COMPACT_MODE_CLUSTER_CPU_COUNT" default:"18"`
+	OCSCompactModeClusterRAMGiB           int64             `envconfig:"OCS_COMPACT_MODE_CLUSTER_RAM_GiB" default:"57"`
+	OCSMinimalModeClusterCPUCount         int64             `envconfig:"OCS_MINIMAL_MODE_CLUSTER_CPU_COUNT" default:"12"`
+	OCSMinimalModeClusterRAMGiB           int64             `envconfig:"OCS_MINIMAL_MODE_CLUSTER_RAM_GiB" default:"57"`
+	OCSStandardModeClusterCPUCount        int64             `envconfig:"OCS_STANDARD_MODE_CLUSTER_CPU_COUNT" default:"24"`
+	OCSStandardModeClusterRAMGiB          int64             `envconfig:"OCS_STANDARD_MODE_CLUSTER_RAM_GiB" default:"57"`
+	OCSRequiredHostMinimalAndStandardMode int64             `envconfig:"OCS_REQUIRED_HOSTS_MINIMAL_AND_STANDARD_MODE" default:"6"`
+	OCSDeploymentType                     ocsDeploymentMode `envconfig:"OCS_DEPLOYMENT_TYPE" default:"None"`
+	OCPMinCPUCoresWorker                  int64             `envconfig:"OCP_MIN_CPU_CORES_WORKER" default:"2"`
```

Comment:
Add a comment that the OCP resource elements are temporary and will be removed once we can query for these values .